containers/init/build/php/build_qps_worker.sh

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+export GRPC_LIB_SUBDIR="libs/opt"
+export CFLAGS="-Wno-parentheses-equality"
+export root="/src/workspace"
+
+make static_c shared_c EMBED_OPENSSL=true EMBED_ZLIB=true
+
+cd src/php/ext/grpc
+
+phpize
+./configure --enable-grpc="/src/workspace" --enable-coverage --enable-tests
+
+cd /src/workspace/src/php/tests/qps
+
+composer install
+
+cd ../../../../third_party/protobuf/php/ext/google/protobuf
+
+phpize
+./configure
+
```

Comment:
@codeblooded Actually I am pretty unsure about his part, I have took the content of their script and put them here, however I am not so sure if any off the lines are redundant, do you have some idea of that?

Reply:
I do not think it is redundant b/c we change directory in line 32. So, they configure grpc in lines 25 and 26 and protobuf in lines 34 and 35. I don't know why they would do grpc before protobuf, but the redundancy isn't wrong.